documentation/modules/deploying/con-deploy-paths.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-tasks.adoc
+
+[id='con-deploy-paths-{context}']
+= Deployment options
+
+[role="_abstract"]
+You can set up a basic deployment with a single Kafka cluster.
+You might use this configuration for development or testing. 
+Or you can use Strimzi to manage a number of Kafka clusters in a production environment. 
+
+The first step for any deployment of Strimzi is to install the Cluster Operator using the `install/cluster-operator` files.
+
+A single command applies all the installation files in the `cluster-operator` folder: `kubectl apply -f ./install/cluster-operator`.
+
+The command sets up everything you need to be able to create and manage a Kafka deployment, including the following:
+
+* Cluster Operator
+* Strimzi CRDs
+* Role bindings
```

Comment:
I would say `RBAC resources`

```suggestion
* RBAC resources
```

Reply:
I wonder if showing the resource/kind names might be useful here (search wise). We mention bindings elsewhere in the docs.

> ...including the following:
> 
> * Cluster Operator (`Deployment`)
> * Strimzi CRDs (`CustomResourceDefinition`)
> * RBAC resources (`ClusterRoleBinding`)
> * Service account (`ServiceAccount`)

Reply:
It might be useful I guess. But then it should be ... 

```
...including the following:

* Cluster Operator (`Deployment`, `ConfigMap`)
* Strimzi CRDs (`CustomResourceDefinition`)
* RBAC resources (`ClusterRole`, `ClusterRoleBinding`, `RoleBinding`)
* Service account (`ServiceAccount`)
```